scripts/init.sh

```diff
@@ -11,11 +11,18 @@ pushd  polkadot-sdk
 git checkout snowbridge
 popd
 popd
-
 ln -sf ../polkadot-sdk polkadot-sdk
 
-echo "Setting up submodules"
-git submodule update --init --recursive || true
+echo "Checkout lodestar Snowfork fork"
+pushd ..
+  if [ ! -d "lodestar" ]; then
+    git clone https://github.com/ChainSafe/lodestar
```

Comment:
```suggestion
    git clone https://github.com/ChainSafe/lodestar
    ln -sf ../lodestar lodestar
```

And remove symlink at line 25, otherwise if there's already a symlink and you run the script it adds another symlink inside the lodestar dir.

Reply:
Nice catch! Fix in https://github.com/Snowfork/snowbridge/pull/1212/commits/fd516ba1dbe3dac660a665fda82e0c85f1b48de3